### DISCUSSION TITLE
表格分多页，调用 reloadData 方法后，数据更新了，但下面的分页栏的当前页没有更新

### DISCUSSION THREAD

**Discussion by HuangLei0x2e5** [layui/layui#2047] [问答]
有多页数据，选中最后一页，然后点击刷新按钮，在按钮事件中调用了table.reloadData方法

调用后，数据更新了，但是下方的分页栏的当前页还是最后一页

刷新前
![before](https://github.com/layui/layui/assets/76526133/73a77be0-0b4a-47ee-96ea-b541a11680b1)

刷新后

![after](https://github.com/layui/layui/assets/76526133/0838e7f8-9508-4626-ac6e-ed4ba505f82c)

**Comment by sentsim** [comment]
```
page: {
  curr: 1
}
```

**Reply by HuangLei0x2e5** [reply]
> ```
> page: {
>   curr: 1
> }
> ```

您好，能否讲的详细一点？我在哪里调用这段代码？

**Reply by LSL1618** [reply]
仔细看文档 [reloadData数据重载](https://layui.dev/docs/2/table/#table.reloadData)

**Reply by sentsim** [reply]
参考这个示例：https://layui.dev/docs/2/table/#demo-search
文档要仔细阅读。
